src/components/RealTimeInterface/PolledThumbnails.vue

```diff
@@ -30,26 +30,24 @@ const getThumbnails = async () => {
     method: 'GET',
     successCallback: (data) => {
       thumbnails.value = data.results
-        .filter(result => result.url.includes('e91-large_thumbnail'))
+        .filter(result => result.url.includes('e01-large_thumbnail'))
         .map(result => result.url)
-      if (thumbnails.value.length > 0) {
-        emits('thumbnailsFetched', thumbnails.value.length)
-      }
+      emits('thumbnailsFetched', thumbnails.value.length)
     },
     failCallback: console.error
   })
 }
 
 onMounted(() => {
   getThumbnails()
-  const intervalMs = configurationStore.demo ? 15000 : 3000
+  const intervalMs = configurationStore.demo ? 15000 : 1000
```

Comment:
Does this mean if its "demo" mode it polls at a much slower rate then if its not demo mode? I would've thought you want the opposite for testing?